workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/NewAppFlow/PublishAppOutlet.tsx

```diff
@@ -105,12 +105,14 @@ const PublishAppOutlet = () => {
 						<pre>
 							{JSON.stringify(
 								(function () {
-									const _context = {...context};
+									const referenceContent: Partial<
+										typeof context
+									> = {...context};
```

Comment:
The type is fine but leave as _context

```suggestion
									const _context: Partial<
										typeof context
									> = {...context};
```